xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/js/xwiki/usersandgroups/usersandgroups.css

```diff
@@ -50,6 +50,10 @@
 #usersandgroupstable.rights .rights {
   width: 15% !important;
 }
+#usersandgroupstable button.rights-edit {
```

Comment:
If this is done for accessibility reason. Can you add a comment explaining what's the impact of this block on accessibility?